### PR TITLE
A few minor updates

### PR DESCRIPTION
**Pull request by Tbenito**

### CODE REVIEW

**Comment by millette** [comment]
Woot, je regarde ça, merci @Tbenito :-)

@ubik23 Je semble avoir les droits pour merger, comme prévu.

**Comment by millette** [comment]
@Tbenito Un truc utile sur github c'est qu'on peut automatiquement fermer les issues quand le PR est accepté. Détails: <https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword>

For example a commit including "Fix #107" would have worked, or better: "Fix #107: del about page, now replaced by a modal" to make it explicit.

No need to change anything now, just saying for next times.

**Comment by millette** [comment]
Note to self: this covers (in order)

- #107
- #103
- #88
- #85
- #104

File: src/index.html [hunk 1]
```diff
@@ -437,15 +437,15 @@ <h3 class="sidebar__title">Customize</h3>
                     <form id="register-form"action="#">
                       <div class="overlay__container overlay__container__form">
                         <label class="overlay__label">Email address</label>
-                        <input class="first-input overlay__input overlay__input__email" type="email" placeholder="Enter Email" name="email" required>
+                        <input class="first-input overlay__input overlay__input__email" type="email" autocomplete="username" placeholder="Enter Email" name="email" required>
```
Comment (millette) [inline src/index.html:440]: Should me "email" - I will fix it right away.

Comment (millette) [reply src/index.html:440]: Done!

**Comment by Tbenito** [comment]
C'est compris, Merci Robin.

**Pull request merged**